src/Microsoft.ML.Data/Evaluators/MultiClassClassifierEvaluator.cs

```diff
@@ -814,10 +814,10 @@ private void CheckInputColumnTypes(Schema schema)
 
             var scoreType = schema[ScoreIndex].Type as VectorType;
             if (scoreType == null || scoreType.Size < 2 || scoreType.ItemType != NumberType.Float)
-                throw Host.Except("Score column '{0}' has type '{1}' but must be a vector of two or more items of type R4", ScoreCol, scoreType);
+                throw Host.Except(nameof(schema), "score", "vector of two or more items of type float", ScoreCol, scoreType.ToString());
             var labelType = schema[LabelIndex].Type;
             if (labelType != NumberType.Float && labelType.GetKeyCount() <= 0)
-                throw Host.Except("Label column '{0}' has type '{1}' but must be a float or a known-cardinality key", LabelCol, labelType);
```

Comment:
>a known-cardinality  [](start = 92, length = 20)

did this get omitted due to the recent changes in the Key type?

Reply:
Yes! there will be no difference between known and unknown cardinality keys.